config/optional/access_unpublished.settings.yml

```diff
@@ -1,4 +1,4 @@
-hash_key: review
+hash_key: auHash
```

Comment:
This is the default `hash_key` setting value provided by the module.  We probably want to omit this change if we have documentation, etc, that mentions the `review` token.